package.json

```diff
@@ -58,12 +59,12 @@
     "lint": "npm-run-all --parallel lint:js lint:sass",
     "lint:js": "eslint src",
     "lint:sass": "sass-lint --verbose --no-exit src/**/*.scss",
-    "test": "npm run build && web-ext run -s dist",
     "watch:js": "onchange -p -v 'src/**/*.js' -- npm run bundle:js",
     "watch:css": "onchange -p -v 'src/**/*.scss' -- npm run bundle:css",
     "watch:assets": "onchange -p -v 'src/{manifest.json,icons/*,popup/*.html}' -- npm run copy:assets",
     "storybook": "start-storybook -s ./dist -p 6006",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook",
+    "test": "npm run lint && echo \"tests TBD\""
```

Comment:
[This bit in the `circle.yml`](https://github.com/bwinton/SnoozeTabs/pull/64/files#diff-29944324a3cbf9f4bd0162dfe3975d88R7) does a build on every CircleCI run. So, that should be covered and will count as a failure overall if it breaks.

I took the packaging out of the `npm run build` command because 

1. `bin/build-addon.sh` will do build plus [signing or packaging as appropriate with the presence of AMO credentials](https://github.com/bwinton/SnoozeTabs/pull/64/files#diff-48f6620f507a81dc48998c8c73f4277eR6) from Jenkins

1. Running it as a temporary add-on in Firefox doesn't actually use the package for anything. 

So, having packaging as part of the build task seemed redundant / unnecessary.